server/grpcservice/util.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package grpcservice
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/CeresDB/ceresdbproto/pkg/commonpb"
+	"github.com/CeresDB/ceresmeta/pkg/coderr"
+	"google.golang.org/grpc"
+)
+
+// GetClientConn returns a gRPC client connection.
+func GetClientConn(ctx context.Context, addr string) (*grpc.ClientConn, error) {
```

Comment:
No need to expose these methods (GetClientConn, OkResponseHeader, ResponseHeader).